src/renderer.rs

```diff
@@ -520,29 +468,29 @@ impl Renderer {
         self.particles.push(system.freeze());
     }
 
-    /// Takes the SpriteSheetBuilder and generates a new SpriteSheet.
+    /// Adds an image as a sprite sheet with the specified tile size.
```

Comment:
The old one wasn't helpfull, but I think you can be more helpful here:
```suggestion
    /// Adds a new SpriteSheet to the registered sprite sheets, using the
    /// image and tile_size.
```
Maybe?

Make the return-type a SpriteSheetID or something. It's hard to keep track of all the types youknow!